meshio/gmsh/_gmsh41.py

```diff
@@ -103,26 +115,40 @@ def read_buffer(f, is_ascii, data_size):
 
 
 def _read_entities(f, is_ascii, data_size):
+    # Read the entity section. Return physical tags of the entities, and (for
+    # entities of dimension > 0) the bounding entities (so points that form
+    # the boundary of a line etc).
+    # Note that the bounding box of the entities is disregarded. Adding this
+    # is not difficult, but for the moment, the entropy of adding more data
+    # does not seem warranted.
+
     fromfile = partial(numpy.fromfile, sep=" " if is_ascii else "")
     c_size_t = _size_type(data_size)
     physical_tags = tuple({} for _ in range(4))  # dims 0, 1, 2, 3
+    bounding_entities = tuple({} for _ in range(4))
```

Comment:
Can be have `({}, {}, {}, {})` here and in the line above?

Reply:
Sure. Will push.